sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerIdentityServiceClient.cs

```diff
@@ -33,6 +33,14 @@ protected ConfidentialLedgerIdentityServiceClient()
         {
         }
 
+        /// <summary> Initializes a new instance of ConfidentialLedgerIdentityServiceClient. </summary>
+        /// <param name="identityServiceUri"> The Identity Service URL, for example https://identity.accledger.azure.com. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="identityServiceUri"/> or <paramref name="credential"/> is null. </exception>
+        public ConfidentialLedgerIdentityServiceClient(Uri identityServiceUri, TokenCredential credential) : this(identityServiceUri, credential, new ConfidentialLedgerClientOptions())
```

Comment:
Same comment as on ConfidentialLedgerClient.

Reply:
This added ctor just needs to be [suppressed ](https://github.com/Azure/azure-sdk-for-net/blob/1c90cb1fa2ff8d43983fee165ab2edbda8e036ea/sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerIdentityServiceClient.cs#L14) like the other ctor was because this client does not require any auth at all. 

Alternatively, if we have a way to selectively set "no-auth" for a single client where the autorest.readme specifies multiple clients, this would address the problem as well.

Reply:
Is the suppression still there in the customization layer, and did not take effect when the clients were regenerated?  If so, that sounds like a bug we should fix in the generator.

Reply:
It is taking effect on the existing ctor, but this version of the generator added an additional one which is not currently being suppressed. 

While this is not a bug, ideally, we'd have a feature to allow this to be configured in the autorest options (generate the client without auth side-by-side with another client that does require auth). However, this is an edge case that doesn't exist in any other clients that I am aware of, so my thought is that suppression is acceptable here.

Reply:
That makes sense.

How is this client constructed?  Do you have samples showing how auth works in the library?  (now I'm curious)

Reply:
discussed offline